coalib/settings/ConfigurationGathering.py

```diff
@@ -132,6 +133,20 @@ def warn_config_absent(sections, argument, log_printer):
     return False
 
 
+def get_default_section(section, sections):
+    default_section = ('.'.join(section.split('.')[:-1])
+                       if unescaped_search_for('.', section) else None)
+
+    if default_section:
+        if default_section in sections:
```

Comment:
I don't think there'd be a section named `""`. So we can safely remove the outer `if`?